set.mm

```diff
@@ -382517,10 +382525,20 @@ proved using the inner form (see theorem 9.6 of [Schwabhauser] p. 69
 
   ${
     $d a b g i j $.
-    $( Define the relation congruence bewteen shapes.  Definition 4.4 of
-       [Schwabhauser] p. 35.  Ideally, we would define this for functions of
-       any set, but we will used words (functions over ` NN ` ) in most cases.
-       (Contributed by Thierry Arnoux, 3-Apr-2019.) $)
+    $( Define the relation congruence between shapes.  Definition 4.4 of
```

Comment:
the relation OF congruence

Reply:
congruence relation?

Reply:
Probably both are correct, but the former has the advantage of having "congruence between shapes" as a substring.

Reply:
Sounds good. I'll use "the relation OF congruence between shapes".